### ISSUE TITLE
New site with Elm Markup build fails (tags?)

### ISSUE THREAD

**Issue by julianpistorius** [alexkorban/elmstatic#22]
When I create a new site using Elm Markup:

```bash
mkdir new-site
cd new-site
elmstatic init --elm-markup
elmstatic build
```

... I get an error about tags:

```
Building the site

Error: Error in _posts/2019-07-29-elmstatic-supports-elm-markup.emu:
Undeclared tags: [s, o, f, t, w, a, r, e, h]
You can declare tags in config.json
    at /Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/bin/elmstatic.js:199:31
    at _map (/Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_map.js:6:19)
    at map (/Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/map.js:64:14)
    at /Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_dispatchable.js:41:15
    at /Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_curry2.js:22:18
    at f1 (/Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_curry1.js:18:17)
    at /Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_pipe.js:3:27
    at /Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/node_modules/ramda/src/internal/_arity.js:10:19
    at generatePostConfigs (/Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/bin/elmstatic.js:219:6)
    at generateEverything (/Users/johndoe/.nvm/versions/node/v12.16.1/lib/node_modules/elmstatic/bin/elmstatic.js:386:22)
```

When I edit `_posts/2019-07-29-elmstatic-supports-elm-markup.emu` and remove the 'software' tag so that the tags line looks like this:

```
|> Metadata
    title = Elmstatic now supports elm-markup
    tags = 
```

... and build it again, it works fine:

```
$ elmstatic build
Building the site
  Compiling layouts
Dependencies ready!           
Success! Compiled 7 modules.

    Tag ─────┬──> elm.js
    Page ────┤
    Post ────┤
    Posts ───┘

  Generating pages and posts
  Cleaning out the output path (_site)
  Writing HTML
  Generating feeds
  Duplicating pages
  Copying resources
```

Note: Also no problem when using Markdown.

**Comment by alexkorban** [comment]
Fixed in v0.7.1, thanks for reporting. It was a regression, but for now I'm mystified as to how it got past the smoke test.

**Comment by julianpistorius** [comment]
Excellent! Thanks for the quick fix.
